tests/diag.sh

```diff
@@ -958,7 +958,7 @@ dep_kafka_url=http://www-us.apache.org/dist/kafka/2.0.0/kafka_2.11-2.0.0.tgz
 dep_kafka_cached_file=$dep_cache_dir/kafka_2.11-2.0.0.tgz
 
 if [ -z "$ES_DOWNLOAD" ]; then
-	export ES_DOWNLOAD=elasticsearch-5.6.9.tar.gz
```

Comment:
do we have a hard reason for the other version or just "this is newer / this is now in widerspread use"? I ask because we intentionally have different versions to cover typical use cases. Also, I cache the ES tarballs inside containers, as somtimes the download takes veeeeery long. So if we change the default, it is wise to update the container def.